pyproject.toml

```diff
@@ -269,6 +269,10 @@ max-returns = 10
 "src/otx/**/*.py" = [
     "ERA001",
 ]
+# See https://github.com/openvinotoolkit/training_extensions/actions/runs/7109500350/job/19354528819?pr=2700
```

Comment:
It seems that there is a mis-rebase, could you check? this line already made from https://github.com/openvinotoolkit/training_extensions/pull/2700/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R272

Reply:
Fixed in https://github.com/openvinotoolkit/training_extensions/pull/2702/commits/03248baeaa957fddbeb21d631d71b9e651f73429